src/availability.rs

```diff
@@ -171,6 +171,26 @@ where
             }
             .boxed()
         })?
+        .get("get_leaf_range", move |req, state| {
+            async move {
+                let from = req.integer_param::<_, usize>("from")?;
+                let until = req.integer_param("until")?;
+
+                state
+                    .get_leaf_range(from..until)
+                    .await
+                    .enumerate()
+                    .then(|(index, fetch)| async move {
+                        fetch.with_timeout(timeout).await.context(FetchLeafSnafu {
+                            resource: (index + from).to_string(),
+                        })
+                    })
+                    .map(|result| result.map(LeafQueryData::from))
```

Comment:
Is there an error if we don't have this `LeafQueryData::from`? I thought `get_leaf_range` is already returning a stream of `LeafQueryData`.

Reply:
No error.